qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -4,6 +4,59 @@
 from qcodes import VisaInstrument
 from qcodes.instrument.parameter import ArrayParameter
 from qcodes.utils.validators import Numbers, Ints, Enum, Strings
+from qcodes.instrument.channel import InstrumentChannel, ChannelList
+
+
+class SR830AuxArray(InstrumentChannel):  # We see that InstrumentChannel, ChannelList are something of a misnomer
+    def __init__(self, parent, name, aux_number):
+        super().__init__(parent, name)
+
+        self.add_parameter('aux_in',
+                           label='Aux input {}'.format(aux_number),
+                           get_cmd='OAUX? {}'.format(aux_number),
+                           get_parser=float,
+                           unit='V')
+
+        self.add_parameter('aux_out',
+                           label='Aux output {}'.format(aux_number),
+                           get_cmd='AUXV? {}'.format(aux_number),
+                           get_parser=float,
+                           set_cmd='AUXV {0}, {{}}'.format(aux_number),
+                           unit='V')
+
+
+class SR830ChannelArray(InstrumentChannel):
+    def __init__(self, parent, name, channel_number):
+        super().__init__(parent, name)
+
+        self._parent = parent
```

Comment:
This should not be needed as its assigned in the super class

Reply:
Oh yeah, I forgot